_posts/en/meetings/2016-07-21-meeting.md

```diff
@@ -0,0 +1,130 @@
+---
+title: IRC meeting summary for 2016-07-21
+permalink: /en/meetings/2016/07/21/
+name: 2016-07-21-meeting
+type: meetings
+layout: page
+lang: en
+version: 1
+---
+{% include _toc.html %}
+ 
+- [Link to this week logs](https://botbot.me/freenode/bitcoin-core-dev/2016-07-21/?msg=70044352&page=2)
+- [Meeting minutes by meetbot](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2016/bitcoin-core-dev.2016-07-21-18.59.html)
+ 
+---
+ 
+## Notes / short topics
+
+- 0.13 has been branched of master since a couple of days.
+- Jeremyrubin has been working on refactoring checkqueue.h including some nice improvements. Cfields has been working on optimizing the sigcache and proposes to work together to come up with a good representative bench for testing improvements.
+- Currently the wallet code is preventing to create outputs below dust by using txminRelayFee. When this was bumped last year in PR[#6793][] some transactions couldn't be relayed anymore causing some stress to users. NicolasDorier is working on avoiding such problems in the future in PR[#8356][].
+
+## Main topics
+ 
+- 0.13.0
+- Remove ISM
+- sigops max size, and the sigops per byte limit
+
+## 0.13.0
+
+### background
+ 
+The Bitcoin Core team is working towards the 0.13.0 release ([full schedule](https://github.com/bitcoin/bitcoin/issues/7679)) and RC1 is [available](https://bitcoin.org/bin/bitcoin-core-0.13.0/test.rc1/) since 2016-07-20.
+
+### meeting comments
+ 
+RC1 got some [feedback](https://github.com/bitcoin/bitcoin/issues/8383) noticing that when encrypting the wallet, it uses the same HD seed, which means the HD seed has been unencrypted on the disk when the wallet was created. Jonasschnelli is working on a [fix][#8389] to create a new HD seed after encrypting the wallet.
+
+A common complaint is the lack of a way to export the HD seed. Jonasschnelli has a [pull request][#8206] which is easy to review and has a low impact that exports the seed to dumpwallet. Importing is a different issue with much more impact as the wallet currently doesn't support multiple seeds. This is a feature for 0.14.
+
+Luke-jr notes the default policy isn't performing as well as inadvisable policies for which he made a [pull request][#8388]. This is a pretty large change and requires some more discussion.
```

Comment:
> the default policy isn't performing as well as inadvisable policies

I'm having difficulty parsing this clause; are there maybe some missing words or extra details you can add?